source/winVersion.py

```diff
@@ -105,18 +105,22 @@ def _getWindowsReleaseName(self) -> str:
 		On server systems, unless noted otherwise, client release names will be returned.
 		For example, 'Windows 10 1809' will be returned on Server 2019 systems.
 		"""
-		if (self.major, self.minor) == (6, 3):
-			return "Windows 8.1"
-		elif self.major == 10:
-			# From Version 1511 (build 10586), release Id/display version comes from Windows Registry.
+		match (self.major, self.minor):

```

Comment:
Nice to see this new structural pattern, and I agree with its use.

However, I remind here that the computation of the complexity of a function does not take it into account unfortunately with our tools; it becomes always more urgent to move to a new linting tool (or version) if we want to continue to check complexity correctly.